chef/cookbooks/nova/recipes/config.rb

```diff
@@ -329,7 +316,6 @@
             :keystone_settings => keystone_settings,
             :cinder_insecure => cinder_insecure || keystone_settings['insecure'],
             :ceph_user => ceph_user,
```

Comment:
So, I agree that problem with `ceph_user` attribute should be fixed with ephemeral rbd support change. I removed `ceph_user` from nova.conf template on @vuntz wish.

Can we continue with merging?